tests/config.json

```diff
@@ -101,8 +101,7 @@
     "input": {
       "tables": [
         {
-          "source": "out.c-main.categories",
-          "destination": "out.c_main.categories.csv"
+          "source": "out.c_main.categories.csv"         
```

Comment:
To `.csv` je tam ale blbost, ne? Viz. https://developers.keboola.com/extend/common-interface/config-file/#input-mapping--basic

Reply:
vtom teste musi byt source zhodne s menom suboru. to ci tam ma nakonci .csv je imho jedno. Co som pochopil z dokumentacie ze ak neni destination tak sa subor vola podla source tj tableId a v tomto pripade tableId obsahuje .csv na konci a tak sa vola aj subor:
https://github.com/keboola/gooddata-writer-v3/blob/master/tests/tables/out.c_main.categories.csv

Ako pises v https://github.com/keboola/gooddata-writer-v3/pull/14#issuecomment-426335902 tak upravim source a meno suboru(odtranim .csv)